devops/windows/run--base-docker-linux/start.ps1

```diff
@@ -1,289 +1,278 @@
-                                             $linker_jar `
-                               -WindowStyle Normal `
-                               -WorkingDirectory $linker_folder `
-                               -Debug `
-                               -Verbose `
-                               -PassThru `
-                               -RedirectStandardError 'linker-stderr.txt'
-#                               -RedirectStandardOutput 'linker-stdout.txt'
-$linker_handle | Export-Clixml -Path (Join-Path './' 'linker_handle.xml')
-
-
-#
-# start api
-#
-if (Test-path $api_folder) {
-  Write-Host ${api_folder}' exists'   
-} else {
-  New-Item $api_folder -ItemType Directory
-  Write-Host 'Folder Created successfully'
-}
-$api_handle = Start-Process -FilePath java `
-                            -ArgumentList $def_api_log4j_level, `
-                                          $def_postgresql_ip, `
-                                          $def_postgresql_port, `
-                                          $def_postgresql_user, `
-                                          $def_postgresql_password, `
-                                          $def_postgresql_notifications_db, `
-                                          $def_kafka_bootstrap_servers, `
-                                          $def_api_kafka_application_id, `
-                                          $def_dgraph_hosts, `
-                                          $def_dgraph_ports, `
-                                          $def_etl_ip, `
-                                          $def_etl_http_port, `
-                                          $def_controller_ip, `
-                                          $def_controller_http_port, `
-                                          $def_linker_ip, `
-                                          $def_linker_http_port, `
-                                          $def_api_ip, `
-                                          $def_api_http_port, `
-                                          '-server', `
-                                          '--enable-preview', `
-                                          $api_jar `
-                            -WindowStyle Normal `
-                            -WorkingDirectory $api_folder `
-                            -Debug `
-                            -Verbose `
-                            -PassThru `
-                            -RedirectStandardError 'api_stderr.txt' 
-#                            -RedirectStandardOutput 'api_stdout.txt'
-$api_handle | Export-Clixml -Path (Join-Path './' 'api_handle.xml')
-
-# PRODUCTION SERVE
-#Push-Location ..\..\..\JeMPI_Apps\JeMPI_UI
-#  # Serve the built project
-#  $serveCommand = "serve -s build -l $serveUIPort"
-#  Invoke-Expression -Command $serveCommand
-#Pop-Location
+$script_path = $MyInvocation.MyCommand.Path
+$script_dir = Split-Path $script_path
+Set-Location $script_dir
+
+. $script_dir\local.ps1
```

Comment:
What is `local.ps1`. Is it auto-generated?

Reply:
local.ps1 is a private script that defines IP address for your setup
contains a single line   eg

$linux_server_ip                               = '192.168.88.252'

this is the linux host address for the windows guest java apps to connect connect to external services - kafka etc.  it's specific for a machine